lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -145,6 +150,19 @@ def close_running_iperf_servers(self):
         )
         return ret_code == 0
 
+    def update_mtu_size(self, set_mtu=False):
+        """ Update MTU size in TRF server """
+        # Set MTU size to 1400 for ipv6
+        if set_mtu == True:
+          ret_code = self.exec_command("sudo /sbin/ifconfig eth3 mtu 1400")
+          if ret_code != 0:
+            return False
+        else :
+          ret_code = self.exec_command("sudo /sbin/ifconfig eth3 mtu 1500")
+          if ret_code != 0:
+            return False
+        return True
```

Comment:
why do you need to change MTU of eth3 device ?
which test cases fails if you dont change MTU?

Reply:
Same as above